### PR TITLE
modify the name and description of Inspur CSI drivers

### PR DESCRIPTION
**Pull request by zhangzhenguang2017**
/release-note-none

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
-[Inspur InStorage CSI](https://github.com/OpenInspur/instorage-k8s) | `csi-instorage` | v1.0 | A Container Storage Interface (CSI) Driver for Inspur AS/HF/CS/CF Series Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
-[Inspur InStorage CSI for Mass Storage](https://github.com/OpenInspur/instorage-k8s) | `csi-instorage` | v1.0 | A Container Storage Interface (CSI) Driver for Inspur Mass Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
+[Inspur InStorage CSI for Primary Storage](https://github.com/OpenInspur/instorage-k8s) | `csi-instorage` | v1.0 | A Container Storage Interface (CSI) Driver for inspur AS/HF/CS/CF Series Primary Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
+[Inspur InStorage CSI for SDS Storage](https://github.com/OpenInspur/instorage-k8s) | `csi-instorage` | v1.0 | A Container Storage Interface (CSI) Driver for inspur AS13000 Series SDS Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
```
Comment (xing-yang) [inline book/src/drivers.md:52]: I looked at https://github.com/OpenInspur/instorage-k8s.  It appears that it is one CSI driver that supports different types of storage systems?  If so, you should combine line 51 and 52 into one entry.  For example, on line 48, IBM block storage driver supports multiple types of IBM storage systems.

Comment (zhangzhenguang2017) [reply book/src/drivers.md]: I have combined line 51 and 52 into one entry.

**Pull request merged**